vyxal/elements.py

```diff
@@ -3078,6 +3078,16 @@ def run_length_decoding(lhs, ctx):
         return LazyList(temp)
 
 
+def shuffle(lhs, ctx):
+    """Element Þ℅
+    (lst) -> Return a random permutation of a
+    """
+
+    temp = deep_copy(lhs)
```

Comment:
Do we really need a *deep* copy? Why not just `lhs[::]` or something?

Reply:
It's just the name - `lhs[::]` doesn't properly copy lazylists